airbyte-cdk/python/airbyte_cdk/base_python/cdk/streams/auth/oauth.py

```diff
@@ -59,9 +59,9 @@ def get_access_token(self):
     def token_has_expired(self) -> bool:
         return pendulum.now() > self._token_expiry_date
 
-    def get_refresh_request_body(self) -> Mapping[str, any]:
+    def get_refresh_request_body(self) -> Mapping[str, Union[str, Any]]:
         """ Override to define additional parameters """
-        payload = {
+        payload: MutableMapping[str, Any] = {
```

Comment:
had to define this as the return type is a `Mapping`; however we are modifying this struct before returning it.